pycode/memilio-simulation/CMakeLists.txt

```diff
@@ -32,6 +32,9 @@ endif()
 # add in C++ library
 add_subdirectory(${CMAKE_CURRENT_SOURCE_DIR}/../../cpp ${CMAKE_CURRENT_BINARY_DIR}/cpp EXCLUDE_FROM_ALL)
 
+# add path to bindings
+include_directories(memilio/simulation/bindings)
```

Comment:
why is that better than the `target_include_directories` you removed?

Reply:
All of the individual targets use this include directories, so it seemed to make sense to set it for all targets once instead of calling the same function multiple times

Reply:
`target_...` functions avoid global declarations that might contaminate other projects (same reason to avoid global variables and declarations in the global namespace). We had problems with this before. I guess it's unlikely this CmakeLists is included somewhere else, so it's a local problem. But in general use the `target_...` versions of cmake functions, unless you are absolutely certain that everyone using the file or adding to it requires the global setting and the normal `target_link_libraries` inheritance doesn't work for some reason.